pkg/sfu/simulcast.go

```diff
@@ -0,0 +1,29 @@
+package sfu
+
+import "time"
+
+const (
+	quarterResolution = "q"
+	halfResolution    = "h"
+	fullResolution    = "f"
+)
+
+type SimulcastConfig struct {
+	BestQualityFirst    bool `mapstructure:"bestqualityfirst"`
+	EnableTemporalLayer bool `mapstructure:"enabletemporallayer"`
+}
+
+type simulcastTrackHelpers struct {
+	switchDelay       time.Time
+	temporalSupported bool
+	temporalEnabled   bool
+	lTSCalc           atomicInt64
```

Comment:
This is the only property we use I think. Probably should consume this into DownTrack or some place more appropriate.